lib/passenger.rb

```diff
@@ -20,5 +20,25 @@ def get_drivers
     def add_trip(trip)
       @trips << trip
     end
+
+    def sum_trip_cost
+      total_cost = 0
+      trips.each do |trip|
+        if trip.cost == nil
+          next
+        else
+          total_cost += trip.cost
+        end
+      end
+      return total_cost
+    end
+    
+    def sum_trip_time
+      subtotal = 0
+      trips.each do |trip|
+          subtotal += trip.duration
+      end
```

Comment:
Looks like you're missing a check for `nil` here. `duration` currently returns `nil` if the `end_time` is `nil`, which will cause this code to raise an error.